src/features/student/additionalSkills/components/interactions/inputs/AdditionalSkillCommentInput/AdditionalSkillCommentInput.test.ts

```diff
@@ -0,0 +1,216 @@
+import { AvInputStub, BddTest } from '@avenirs-esr/avenirs-dsav/test-utils'
+import { mount, type VueWrapper } from '@vue/test-utils'
+import { beforeEach, expect, vi } from 'vitest'
+import AdditionalSkillCommentInput from './AdditionalSkillCommentInput.vue'
+
+BddTest().given('an additional skill comment input', () => {
+  let wrapper: VueWrapper<InstanceType<typeof AdditionalSkillCommentInput>>
+
+  const stubs = {
+    AvInput: AvInputStub
+  }
+
+  BddTest().when('the component is mounted with default props', () => {
+    beforeEach(() => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        global: { stubs }
+      })
+    })
+
+    BddTest().then('it should render the AvInput component', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.exists()).toBe(true)
+    })
+
+    BddTest().then('it should have default isValid prop set to false', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('isValid')).toBe(false)
+    })
+
+    BddTest().then('it should have default isTextarea prop set to true', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('isTextarea')).toBe(true)
+    })
+
+    BddTest().then('it should have default labelVisible prop set to true', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('labelVisible')).toBe(true)
+    })
+
+    BddTest().then('it should have default disabled prop set to false', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('disabled')).toBe(false)
+    })
+
+    BddTest().then('it should have default required prop set to true', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('required')).toBe(true)
+    })
+
+    BddTest().then('it should have default maxlength prop set to 400', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('maxlength')).toBe(400)
+    })
+
+    BddTest().then('it should use default i18n label', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('label')).toBe('Description de ma compétence')
+    })
+  })
+
+  BddTest().and('the component is mounted with custom props', () => {
```

Comment:
```suggestion
  BddTest().when('the component is mounted with custom props', () => {
```

S'il n'y a pas de when avant un then, alors ce and doit être un when